fastlane/README.md

```diff
@@ -0,0 +1,38 @@
+fastlane documentation
```

Comment:
Is this file really necessary?

Reply:
Not really. I kept it around since I saw other projects keeping it.
I can add it to .gitignore.